coderd/database/queries/workspaces.sql

```diff
@@ -914,3 +914,23 @@ SET
 	user_acl = @user_acl
 WHERE
 	id = @id;
+
+-- name: GetRegularWorkspaceCreateMetrics :many
+SELECT
+	t.name AS template_name,
+	COALESCE(tvp.name, '') AS preset_name,
+	o.name AS organization_name,
+	COUNT(*) AS created_count
+FROM workspaces w
+INNER JOIN workspace_builds wb ON wb.workspace_id = w.id
+	AND wb.build_number = 1
```

Comment:
This metric counts workspaces whose first build succeeded. If the first build failed and a later build succeeded, that workspace is not included. We could change the query to count the first successful build instead; however, we would still need to exclude prebuild-created workspaces by verifying that the first build was not initiated by the prebuilds system user. 
It is guaranteed that the first build of a prebuilt workspace will always be initiated by the prebuilds system user, right?

Reply:
> Is this possible? 

Yes, it's possible and supported to create a workspace for another user:

```
$ coder create --help                          
coder v2.25.1-devel+dd867bd74

USAGE:
  coder create [flags] [workspace]

  Create a workspace

    - Create a workspace for another user (if you have permission):
  
        $ coder create <username>/<workspace_name> 
```

> If yes, should we allow this?

Good question :) It gets very weird to reason about especially if the workspace that gets created is a prebuilt workspace...

<img width="1892" height="1050" alt="Screenshot 2025-08-25 at 20 21 35" src="https://github.com/user-attachments/assets/6ca81c9e-e1a2-4b2a-8c5c-ec1d393bc42e" />

I would consider it out of scope of this issue.